docs/libs/ml/index.md

```diff
@@ -30,7 +122,7 @@ under the License.
 </dependency>
 {% endhighlight %}
 
-## Algorithms
+## Algorithm Documentation
```

Comment:
Yeah this becomes a bit problematic now. This text is not really what you expect from the index of a library, so maybe it should be moved to its own section (Vision and roadmap?) with only a very brief introduction remaining on the index.